intranet/static/js/eighth/schedule.js

```diff
@@ -175,5 +175,20 @@ $(function() {
         if (activities !== "" && !confirm("Are you sure you want to add the following activities without a sponsor?\n" + activities)) {
             e.preventDefault();
         }
+        $("tr.form-row:not(.hidden)").each(function(i, el) {
+          if($(this).hasClass("scheduled")){
+              var stickyCancel = "";
```

Comment:
You need to move this outside of this callback function so it's visible in the `confirm()` code.